bindata/v3.11.0/kube-apiserver/defaultconfig.yaml

```diff
@@ -108,7 +108,6 @@ storageConfig:
   ca: /etc/kubernetes/static-pod-resources/configmaps/etcd-serving-ca/ca-bundle.crt
   certFile: /etc/kubernetes/static-pod-resources/secrets/etcd-client/tls.crt
   keyFile: /etc/kubernetes/static-pod-resources/secrets/etcd-client/tls.key
-  storagePrefix: openshift.io
```

Comment:
will we remove this field entirely from storage config in a follow-on?

it seems we always hard-code this 
https://github.com/openshift/origin/blob/1a61ff214ee8a193e4b7331efe8ce6793b3725ec/pkg/cmd/openshift-kube-apiserver/configdefault/openshiftcontrolplane_default.go#L9

Reply:
@derekwaynecarr we can remove this in openshift apiserver operator config